config/jobs/testing/testing-update-prow-periodic.yaml

```diff
@@ -0,0 +1,65 @@
+---
+periodics:
+  # Change the cron string to the update frequency/schedule you prefer. Note that
+  # updates to the bump PR do NOT trigger notifications so long as the repo's
+  # presubmits are not flaky. Updating the PR more frequently then the actual bump
+  # frequency is not a problem and can be convenient.
+  - cron: "05 15-23 * * 1-5"  # Run at 7:05-15:05 PST (15:05 UTC) Mon-Fri
+    name: ci-prow-autobump
+    # Only run this in a build cluster where you feel comfortable storing your GH
+    # bot token.
+    decorate: true
+    # Check out the repo containing the config and deployment files for your Prow instance.
+    extra_refs:
+      - org: pusher
+        repo: testing
+        base_ref: master
+        path_alias: github.com/pusher/testing
+    spec:
+      securityContext:
+        runAsUser: 0
+        runAsGroup: 0
+      containers:
+        - image: quay.io/pusher/autobump:latest  # TODO: add a tag once we've push this the first time.
```

Comment:
Do we have a tag for this yet?

Reply:
No, Just latest

Reply:
I could make one? I'm not envisioning making any changes to the image any time soon though..

Reply:
`:latest` is another word for fail

Reply:
> I could make one? I'm not envisioning making any changes to the image any time soon though..

Please. Something like `vYYYMMDD-GIT_SHORT_SHA` will work just fine.

Reply:
Or maybe even include hour and minute as well.

Reply:
Done! `20200303-37e629a6a`